### PR TITLE
Document the 110 pod limit

### LINKED ISSUE #196: Document max_pods limit of 110
We have a hardcoded limit of 110 pods in the agent code.
We decided to not make this configurable at the moment but we do need to document this limit.

### PR DESCRIPTION
**Pull request by lfrancke**
/closes #196

### CODE REVIEW

File: docs/modules/ROOT/pages/limitations.adoc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+= Limitations
+
+The Stackable Agent has limitations, some of which are documented here.
+
+* The maximum number of Pods supported by the Agent is currently hardcoded to _110_.
+This restriction can be lifted if the need arises.
```
Comment (siegfriedweber) [inline docs/modules/ROOT/pages/limitations.adoc:6]: "Pods" should be lowercase. The other changes are a matter of taste:

```suggestion
* The maximum number of pods supported by the agent is currently hardcoded to 110.
  This restriction can be lifted if the need arises.
```

**Review by siegfriedweber — DISMISSED** [review]
Approved with a minor remark. This change could also be mentioned in the changelog if you want.

**Comment by lfrancke** [comment]
I thought about mentioning it in the changelog but the documentation is not really tied to releases right now. Once we merge this PR the docs will be live (more or less)....I don't have a strong opinion though.

What's your preference?

**Review by siegfriedweber — APPROVED** [review]
I think it is not necessary to mention this pull request in the changelog.

**Pull request merged**